src/model.js

```diff
@@ -41,8 +41,10 @@ const itemValidationModel = function () {
     }
 }
 
-export  {
-            itemsCreationModel, itemValidationModel,
-            tasksModelFactory, listsModelFactory
+module.exports = itemsCreationModel;
+
+// export  {
+//             itemsCreationModel, itemValidationModel,
+//             tasksModelFactory, listsModelFactory
         
-        }
\ No newline at end of file
+//         }
```

Comment:
Add a new line at the end of each file is some kind of a standard. I know I also added no new line when I wrote this file. =) I came across this just recently.

Reply:
I don't see any advantage of adding a new line at the end.